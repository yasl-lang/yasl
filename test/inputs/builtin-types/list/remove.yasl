
const ls = [ .a, .b, .c, .d, .e, .f, .g ]

ls->remove(.c)

echo ls

ls->remove(.g)

echo ls

ls->remove(.a)

echo ls
