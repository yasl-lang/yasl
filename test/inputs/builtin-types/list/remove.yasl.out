[a, b, d, e, f, g]
[a, b, d, e, f]
[b, d, e, f]
